app/components/windows/OneOffWindow.vue

```diff
@@ -1,12 +1,12 @@
 <template>
 <div style="height: 100%">
   <title-bar :title="options.title" class="child-window-titlebar" v-if="!options.isFullScreen" />
-  <div class="blank-slate" v-if="!options.hideBlankSlate">
+  <div v-bind:class="['blank-slate', {'no-titlebar': options.isFullScreen}]" v-if="!options.hideBlankSlate">
```

Comment:
FYI: 省略記法があります https://jp.vuejs.org/v2/guide/syntax.html#v-bind-%E7%9C%81%E7%95%A5%E8%A8%98%E6%B3%95
```suggestion
  <div :class="['blank-slate', {'no-titlebar': options.isFullScreen}]" v-if="!options.hideBlankSlate">
```